cli/args/flags.rs

```diff
@@ -908,6 +909,20 @@ fn compile_subcommand<'a>() -> Command<'a> {
   runtime_args(Command::new("compile"), true, false)
     .trailing_var_arg(true)
     .arg(script_arg().required(true))
+    .arg(
+      Arg::new("side-module")
```

Comment:
Was flag name settled? Me and @aapoalas suggested `--import` which is less colloquial than `--side-module`.

Reply:
No - it's still up to debate/bike-shedding, but I'm fine changing the name in a follow up. I'm not sure I like `--import` better than `--side-module`, seems more ambigious.

Reply:
`--include` ? Shorter is better here...

Reply:
I'm fine with `--include` too

Reply:
Yeah `--include` was our next one https://github.com/denoland/deno/pull/17663#issuecomment-1427085167, sounds perfect if it might include assets as well.

Reply:
ok let's switch to `--include`